docs/DeveloperGuide.md

```diff
@@ -266,31 +266,154 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `GrAB3` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+#### Task Schedule
+
+**UC01: Add a task to the task schedule**
+
+**Preconditions: A task schedule must exist**
+
+**Guarantees: A new task will be added to the task schedule** 
+
+**MSS**
+
+1.  User requests to add a new task into the existing task schedule.
+2.  GrAB3 shows the updated task schedule.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. Task description is empty.
+
+    * 1a1. GrAB3 shows an error message.
+    
+        Use case resumes at step 2.
+
+**UC02: Delete a task**
+
+**Preconditions: task schedule must exist**
+
+**Guarantees: The task specified by the user will be deleted**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to list all tasks for the specified week.
+2.  GrAB3 displays a list of tasks for that week.
+3.  User requests to delete a specific task from the list generated.
+4.  GrAB3 shows the updated list after deletion of the task.
 
     Use case ends.
 
 **Extensions**
 
```

Comment:
Should we add in a case for invalid week specified by the user as an extension to [1]? Like week 123 or week -1. Same thing for UC03 - editing a task.